.changes/app-dirs-api.md

```diff
@@ -0,0 +1,5 @@
+---
+"tauri": minor
+---
+
+Add new app-specific directory APIs and deprecate existing `app_dir` and `log_dir` APIs.
```

Comment:
```suggestion
---
"tauri": minor
"api": minor
---

Add new app-specific `BaseDirectory` enum variants `AppConfig`, `AppData`, `AppLocalData`, `AppCache` and `AppLog` along with equivalent functions in `path` module and deprecated ambiguous variants `Log` and `App` along with their equivalent functions in `path` module.
```

Reply:
I considered adding all the details but wasn't sure if the message could be that long. Good to know it can be!